draft-ietf-quic-qlog-h3-events.md

```diff
@@ -166,6 +166,16 @@ layer, and there isn't always a one to one mapping between HTTP and QUIC events.
 The exchange of data between the HTTP and QUIC layer is logged via the
 "stream_data_moved" and "datagram_data_moved" events in {{QLOG-QUIC}}.
 
+HTTP/3 frames are transmitted on QUIC streams, which allows them to span
+multiple QUIC packets. Some implementations might send a single large frame,
+rather than a sequence of smaller frames, in order to amortize frame header
```

Comment:
Do we need to specify that this only applies to DATA frames?

Reply:
Ah I see the disjoint in my previous understanding of why the text was the way it was. In theory extension frames could do something similar but nothing concrete is in scope, only DATA benefits from that usage.

Frames can span packets and there might be some value is explaining that here. Let me try to tweak the text a bit more.

Reply:
I actually think the text might be fine as-is? There's also the case where you have the H3 frame headers in Packet N and the actual payload in Packet N+1 (say you have multiple streams being packed into the same packet for some reason), so not tying this down to `DATA` only seems fine.

If you really want, you could change it to `Additionally, some implementations might send a single large frame,
rather than a sequence of smaller frames...`, so it's a bit clearer that the "single large frame" is a common occurrence of this issue but not the sole reason :)